pkg/fdbadminclient/admin_client.go

```diff
@@ -82,7 +82,7 @@ type AdminClient interface {
 	ResumeBackups() error
 
 	// ModifyBackup modifies the configuration of the backup.
-	ModifyBackup(int) error
+	ModifyBackup(snapshotPeriodSeconds int, url string) error
```

Comment:
I think passing down a pointer to the `FoundationDBBackup` struct will be more flexible in the future and we need to break the interface only once.